### DISCUSSION TITLE
Project Orang Gw Kerjain! - Tulisan Programmer Gadungan

### DISCUSSION THREAD

**Discussion by giscus** [topidesta/topidesta#27] [Comments]
# Project Orang Gw Kerjain! - Tulisan Programmer Gadungan

Bismillah ... Sebetulnya gw lagi Gabut banget, gak semanget tuh dikantor juga (gegara THR) dan dapet penawaran kerjaan Remote 3x tapi belum…

https://topidesta.my.id/project-orang-gw-kerjain/

**Comment by mdestafadilah** [comment]
gak jadi di kerjain. wkkwk.
